nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/RunMongoAggregation.java

```diff
@@ -73,11 +78,17 @@
 
     static final List<Bson> buildAggregationQuery(String query) throws IOException {
         List<Bson> result = new ArrayList<>();
+        JSONParser jsonParser = new JSONParser();
```

Comment:
What's the reason for using this vs Jackson? We try to avoid personal preference in favor of Jackson wherever possible. Is there a deficiency in Jackson?

Reply:
I don't think so. It was just the first I've found.